Cargo.toml

```diff
@@ -39,3 +39,6 @@ libc = { version = "0.2", optional = true }
 
 [target.'cfg(windows)'.dependencies]
 winapi = { version = "0.3", features = ["minwinbase", "minwindef", "timezoneapi"], optional = true }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+stdweb = "*"
```

Comment:
This should use a standard version number, not a glob import. Otherwise, breakage could happen at any time with an upstream change. Using `default_features = false` would also be a good idea if possible.